corehq/util/log.py

```diff
@@ -58,3 +59,15 @@ def format_details(self, details):
         if details:
             formatted = '\n'.join('{item[0]}: {item[1]}'.format(item=item) for item in details.items())
             return 'Details:\n{}'.format(formatted)
+
+
+class SensitiveErrorMail(ErrorMail):
+    """
+    Extends Celery's ErrorMail class to prevents task args and kwargs from being printed in error emails.
+    """
+    replacement = '(excluded due to sensitive nature)'
+
+    def format_body(self, context):
+        context['args'] = self.replacement
+        context['kwargs'] = self.replacement
```

Comment:
See https://github.com/celery/celery/blob/3.0/celery/worker/job.py#L399 for more context